test/integration/capybara/lurker_test.rb

```diff
@@ -205,6 +205,21 @@ def test_search
            "Found these: #{labels.inspect}")
   end
 
+  def test_search_from_obs_needing_ids
+    login
+
+    visit("/observations/identify")
+    # Search for a location.
+    place = "Massachusetts"
+    fill_in("search_pattern", with: place)
+    click_button("Search")
+    assert_match(/#{:obs_needing_id.t}/, page.title, "Wrong page")
```

Comment:
If I replicate this test manually:
- Go to http://localhost:3000/observations/identify
- Fill in Search box with "Massachusetts"
- Click Search
I end up at `Observations Matching ‘Massachusetts’` rather than  `Observations Needing Identification`

(Is this what's supposed to happen?:
Starting on /observations/identify, I can perform an Observation pattern search, 
which gets me Observations needing ID's limited by that search.
That would be incredibly useful!)

Reply:
Joe - I believe your local server is almost certainly running the wrong version. This works for me locally.

Maybe something's amiss with your version control client? Or, you may just need to `Vagrant reload` and reboot the server. Are you still using VSCode for GitHub? 
Just asking because this seems to have happened a lot recently.